vcxagency-node/src/service/entities/agent/agent.js

```diff
@@ -151,23 +155,33 @@ async function buildAgentAO (entityRecord, serviceWallets, serviceStorage, route
     const fwd = msgObject['@fwd']
     const msg = msgObject['@msg']
     if (!fwd) {
-      throw Error(`'VCX V2 Forward message expected to have field '@fwd'. Message: ${JSON.stringify(msgObject)}'`)
+      throw Error(`${whoami} VCX V2 Forward message expected to have field '@fwd'. Message: ${JSON.stringify(msgObject)}'`)
     }
     if (!msg) {
-      throw Error(`'VCX V2 Forward message expected to have field '@msg'. Message: ${JSON.stringify(msgObject)}'`)
+      throw Error(`${whoami} VCX V2 Forward message expected to have field '@msg'. Message: ${JSON.stringify(msgObject)}'`)
     }
     return router.routeMsg(fwd, Buffer.from(JSON.stringify(msg)))
   }
 
+  async function _handleUpdateComMethod (msgObject) {
+    const { comMethod: { value, type, id } } = msgObject
+    if (type === '2') {
+      await setWebhook(value)
+      return buildMsgCommMethodUpdated(id) // TODO: What's meaning of 'id' field? This reflects dummy-cloud-agency impl.
```

Comment:
It's good to refer there, but I think the function definition is a better place for it than this one.

Reply:
I found it important to note that this particular method reflects dummy cloud agency behaviour - it takes received `id` and put it into `id` in response as well.